### DISCUSSION TITLE
Empty Scan results for opensource Github repo.

### DISCUSSION THREAD

**Discussion by mayank99-99** [aquasecurity/trivy#7265] [Q&A]
### Question

I am trying to scan an opensource github repo (https://github.com/TurboVNC/turbovnc) but it gives me no scan results and is also not able to detect any code files. To verify that I have also tried scanning a specific file from the repository on my local system with `--debug` option enabled.

The commands I have been using:
`./trivy.exe fs .`
`./trivy.exe repo https://github.com/TurboVNC/turbovnc`
`./trivy.exe fs /path/to/project`

Any help or guidance is appreciated.

### Target

Git Repository

### Scanner

Vulnerability

### Output Format

None

### Mode

None

### Version

```bash
Version: 0.53.0
Vulnerability DB:
  Version: 2
  UpdatedAt: 2024-07-30 18:11:14.280959839 +0000 UTC
  NextUpdate: 2024-07-31 00:11:14.280959548 +0000 UTC
  DownloadedAt: 2024-07-30 18:34:21.4916544 +0000 UTC
Check Bundle:
  Digest: sha256:ef2d9ad4fce0f933b20a662004d7e55bf200987c180e7f2cd531af631f408bb3
  DownloadedAt: 2024-07-25 18:53:16.536347 +0000 UTC
```

**Comment by afdesk** [comment]
hi @mayank99-99 
I took a look at the repository. and it seems there are no any files for Trivy scans.

which files do you want to scan inside TurboVNC?

you can read [here](https://aquasecurity.github.io/trivy/v0.53/docs/coverage/language/) about Trivy supported languages.

**Comment by mayank99-99** [comment]
Hello @afdesk 

Thank you for your response. So just to clarify, it will only scan the files/languages those mentioned on the link you provided in the previous comment.

In this case, I thought I might be able to scan .java or .c files as those are part of codebase. Is that not the case? Apologies in advance, if I misinterpreted something.

Best,

**Reply by afdesk** [reply]
Hi @mayank99-99!
yes, you're right. Trivy is designed to scan [Language-specific Packages](https://aquasecurity.github.io/trivy/v0.53/docs/coverage/language/), rather than specific .java or .c files
